### DISCUSSION TITLE
Junctek kg140f

### DISCUSSION THREAD

**Discussion by eliesoldini** [fl4p/batmon-ha#199] [Ideas]
I tried to communicate using dummy with junctek kg140f and i managed to receive many numbers that doesn't make any sense. I am sure lots of junctek users love to retrieve data using batmon.
Is there a chance to integrate junctek in the future?
Can someone help me figure out how to filter and fix the numbers to match what junctek is showing on the screen as i have the shunt and batmon installed, maybe i can contribute to this wonderful program.
Thanks

**Comment by fl4p** [comment]
The dummy just generates random data for testing.

**Reply by Arnix11** [reply]
Can You help us to use batmon with junctek kg-f, kh-f please?

**Comment by chukaonline** [comment]
It would be great if you could add  junctek kg-f, kh-f.
